### PR TITLE
Add platform as an argument to update script

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: tools/update.sh [hunk 1]
```diff
@@ -37,15 +41,35 @@ parse_opts ()
         exit 1
     fi
 
+    if [ "${OPT_OS}" == "" ]
+    then
+        echo "You must specify a value for os."
```
Comment (bhariharan) [inline tools/update.sh]: Can we default to both if not specified?

**Comment by wmathurin** [comment]
👍

**Comment by bhariharan** [comment]
Looks good. 👍

File: tools/update.sh (hunk 1 above)
Comment (kchitalia) [inline tools/update.sh]: Sure. I'll make that change

File: tools/update.sh [hunk 2]
```diff
@@ -37,15 +41,33 @@ parse_opts ()
         exit 1
     fi
 
+    if [ "${OPT_OS}" == "" ]
+    then
+        OPT_OS="all"
+    fi
```
Comment (kchitalia) [inline tools/update.sh]: Done. Changed default option to "all".

**Pull request merged**